subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/auth2/AuthClient.kt

```diff
@@ -183,8 +191,12 @@ class AuthClientImpl @Inject constructor(
         )
     }
 
-    override suspend fun getJwks(): String =
-        authService.jwks().string()
+    override suspend fun getJwks(): String {
```

Comment:
Q: I have not enough context here. About caching this for 1h, checking if there's any risk of doing that, should we worry about concurrency here, should we consider adding an specific FF for this change, etc.

Reply:
I've considered concurrency here and I don't think its a concern. Yes, there is a (small) chance that we'll make more requests to fetch jwks than necessary when/if this method is called by multiple coroutines in parallel, but given the fact this method will not be called often, the impact on overall efficiency should be negligible. The important thing is that there is no potential correctness/consistency issue here, given the nature of the cached data. Caching the token for 1 hour is actually recommended by the API documentation - I'll provide more details in the asana task.